fetch/main.ts

```diff
@@ -0,0 +1,47 @@
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
+
+interface Pokemon {
+  name: string;
+  height: number;
+  weight: number;
+}
+
+async function fetchUser(): Promise<void> {
+  try {
+    const fetchResponse = await fetch(
+      'https://jsonplaceholder.typicode.com/users'
+    );
+    if (!fetchResponse.ok) {
+      throw new Error(`HTTP Error! Status: ${fetchResponse}`);
```

Comment:
This will use `fetchResponse.toString()` in the message, which won't have the status. You should use `fetchResponse.status` instead.